pandas/core/missing.py

```diff
@@ -447,7 +452,6 @@ def wrapper(arr, mask, limit=None):
 
 
 def pad_1d(values, limit=None, mask=None, dtype=None):
-
```

Comment:
normally don't like to edit think not-associated with the PR (e.g. you may have some editor setting which change this)...no big deal